README.md

```diff
@@ -79,3 +79,125 @@ solver: str
 Thoth investigator checks Thoth knowledge Graph and decides which workflows need to be scheduled to increase Thoth knowledge:
 
 - [Security Indicator](https://github.com/thoth-station/si-aggregator) workflow
+
+#### AdviseJustificationMessage
+
+- [AdviseJustificationMessage](https://github.com/thoth-station/messaging/blob/master/thoth/messaging/advise_justification.py)
+
+This message received contains:
+
+```python
+message: str
+justification_type: str
+count: int
+```
+
+These messages get processed to update advise justification metrics (`advise_justification_type_number`)
+
+#### HashMismatchMessage
+
+- [HashMismatchMessage](https://github.com/thoth-station/messaging/blob/master/thoth/messaging/hash_mismatch.py)
+
+This message received contains:
+
+```python
```

Comment:
Don't we add component name e service version also?

Reply:
++